monitor-payments/index.js

```diff
@@ -0,0 +1,40 @@
+const http = require('http');
+const algosdk = require('algosdk');
+
+const { indexer_token, indexer_server, indexer_port, address } = require('./config');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+// client for indexerV2
+const indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Monitoring payments in server using algorand indexer');
+});
+
+server.listen(port, hostname, async () => {
+  console.log(`Server running at http://${hostname}:${port}/`);
+  console.log(`Monitoring payments to address:${address} using algorand-indexer`);
+
+  let currRound = 1;
+  setInterval(async () => {
+    const health = await indexerClient.makeHealthCheck().do();
+    if (currRound !== health.round) {
+      currRound = health.round;
+      let response = await indexerClient.searchForTransactions()
+        .address(address).txType('pay').round(currRound)
+        .do();
```

Comment:
so we need to do pooling? is there a way to subscribe to some events?

Reply:
You mean like a `webhook`? I'll ask

Reply:
[https://discord.com/channels/491256308461207573/813198189334888469/877373710024200222](https://discord.com/channels/491256308461207573/813198189334888469/877373710024200222)

Reply:
Let's add the following comment:
```
// Here we are scanning all payment transactions.
// For optimal result, if you have a control of the indexer PostgreSQL, you should 
// create a trigger with COPY statement to write a log. 
```

Reply:
ok